src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -28,18 +30,23 @@ export const styles = StyleSheet.create({
     flexDirection: 'column',
     height: height - 85
   },
+  flatlistContainer: {
+    paddingTop: 5,
+    paddingBottom: 20
+  },
   chatContainer: {
     display: 'flex',
     height: '100%'
   },
   inputContainer: {
     backgroundColor: colors.white,
-    width: '100%',
-    height: 50,
     zIndex: 100,
     padding: 8,
     borderTopColor: colors.lightgrey,
-    borderTopWidth: 1
+    borderTopWidth: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new styles added to `flatlistContainer` and `inputContainer` seem fine. However, the removal of `width: '100%', height: 50,` from `inputContainer` might affect the layout if these properties were important for the correct display of the component. Please ensure that this change does not negatively impact the UI.



<!-- This is an auto-generated comment by OSS CodeRabbit -->